test/functional/mempool_reorg.py

```diff
@@ -106,8 +106,45 @@ def run_test(self):
 
         self.log.info("Check that the mempool is empty")
         assert_equal(set(self.nodes[0].getrawmempool()), set())
+        assert_equal(set(self.nodes[1].getrawmempool()), set())
```

Comment:
nit: Why is this needed? The `self.sync_all()` in the next line should take care of that already. I think it should be fine to remove either this line or the next.

Reply:
removed